src/Illuminate/Database/Eloquent/Relations/Relation.php

```diff
@@ -376,6 +383,16 @@ protected function whereInMethod(Model $model, $key)
                         : 'whereIn';
     }
 
+    /**
+     * Indicate that the basename of classes should be used when determining morphed class names.
```

Comment:
Small typo:

```suggestion
     * Indicates that the basename of classes should be used when determining morphed class names.
```

Reply:
Not a typo.

Reply:
I'm not a native English speaker, but then how does it defer from (I logically suppose it's either one or the other, but I understand my comment doesn't add any value whatsoever :) )

https://github.com/laravel/framework/pull/38451/files#diff-770e6afc4f120d519e68c9f0b8de5fc5694eb0bc558b52f7464db8d9bf10e50fR54